__DEFINES/stylesheet.dm

```diff
@@ -89,6 +90,7 @@ h1.alert, h2.alert		{color: #000000;}
 .borer					{color: #800080; font-weight: bold;}
 .grue					{color: #272728; font-weight: bold;}
 .valentines				{color: #db5b8c; font-weight: bold; font-family: Comic Sans MS, Comic Sans, cursive;}
+.spoken_into_telephone	{color: #330088;}
```

Comment:
Kinda close to the #171717 used in darkmode, try making it brighter for darkmode, please.
This is how it looks: 
![image](https://github.com/vgstation-coders/vgstation13/assets/67024428/fe3456fe-b4ed-4ea2-811c-18381bcfbc97)
Could change it to #aa33ff, which would look like this
![image](https://github.com/vgstation-coders/vgstation13/assets/67024428/b208269d-683e-4a83-85b2-1da0346d11f0)
or #77aa00, which would look like this
![image](https://github.com/vgstation-coders/vgstation13/assets/67024428/fe6d8555-1042-4134-8524-573e5b77a220)

Reply:
went with 77aa00